README.md

```diff
@@ -155,17 +139,17 @@ In the event of a disk failure hosting the PostgreSQL database, the procedure to
 If you wish to stop automatically backing up and optimizing a particular database, you can run `macos-uninstall.sh`:
 
 ```
-% sudo ./macos-uninstall.sh
+sudo sh macos-uninstall.sh
 ```
 
 The script will ask you what database you want to stop backing up and optimizing. The database you specify will then stop being backed up, stop being optimized, and all relevant files will be safely and cleanly removed from your system. The database itself will remain untouched.
 
-### Uninstall on CentOS
+### Uninstall on Red Hat Enterprise Linux
 
-If you wish to stop automatically backing up and optimizing a particular database, you can run `centos-uninstall.sh`:
+If you wish to stop automatically backing up and optimizing a particular database, you can run `enterprise-linux-uninstall.sh`:
 
 ```
-$ sudo ./centos-uninstall.sh
+sudo sh enterprise-linux-uninstall.sh
 ```
 
 The script will ask you what database you want to stop backing up and optimizing. The database you specify will then stop being backed up, stop being optimized, and all relevant files will be safely and cleanly removed from your system. The database itself will remain untouched.
```

Comment:
Add information about the backup files themselves also not being touched. Something to the effect of: The database itself, as well as the backup files that have already been generated, will remain untouched.